zipkin-local.js

```diff
@@ -0,0 +1,129 @@
+"use strict";
+const _interopRequireDefault = require("@babel/runtime/helpers/interopRequireDefault");
+const {
+  THEME_OPTIONS,
+} = require("gatsby-plugin-newrelic-test/utils/constants");
+const { getCiData } = require("./utils/getCiData");
+const ciAttributes = getCiData();
+exports.__esModule = true;
+exports.stop = exports.create = void 0;
+
+const _zipkin = _interopRequireDefault(require("zipkin"));
+
+const _zipkinTransportHttp = require("zipkin-transport-newrelic");
+
+const _zipkinJavascriptOpentracing = _interopRequireDefault(
+  require("zipkin-javascript-opentracing")
+);
+const _nodeFetch = _interopRequireDefault(require("node-fetch"));
+let logger;
+let recorder;
```

Comment:
Nitpick: I don't know if we need to add these as "global" variables. In order to promote [function purity](https://betterprogramming.pub/what-is-a-pure-function-3b4af9352f6f) we should try to avoid functions setting / getting variables that are defined outside their scope.

```suggestion
```

Reply:
Both logger and recorder are used inside other functions so they need to be set globally, unless there's a better way